### PR TITLE
DOC: Adding import numpy to several files

### PR DESCRIPTION
**Pull request by njj67229**
#### Reference issue
See #13049

#### What does this implement/fix?
Adds import numpy as np to examples where necessary 

#### Additional information
Not closed

### CODE REVIEW

**Comment by njj67229** [comment]
Theres an issue in the tests for _util.py that I am looking at now

File: scipy/cluster/vq.py [hunk 1]
```diff
@@ -186,13 +187,13 @@ def vq(obs, code_book, check_finite=True):
...
-    >>> from numpy import array
+    >>> import numpy as np
     >>> from scipy.cluster.vq import vq
-    >>> code_book = array([[1.,1.,1.],
-    ...                    [2.,2.,2.]])
-    >>> features  = array([[  1.9,2.3,1.7],
-    ...                    [  1.5,2.5,2.2],
-    ...                    [  0.8,0.6,1.7]])
+    >>> code_book = np.array([[1.,1.,1.],
+    ...                      [2.,2.,2.]])
```
Comment (tupui) [inline scipy/cluster/vq.py:193]:
```suggestion
    ...                       [2.,2.,2.]])
```

File: scipy/cluster/vq.py [hunk 2]
```diff
@@ -281,14 +282,14 @@ def _kmeans(obs, guess, thresh=1e-5):
...
-    ...                    [ 1.5,2.5],
-    ...                    [ 0.8,0.6],
-    ...                    [ 0.4,1.8],
-    ...                    [ 1.0,1.0]])
-    >>> book = array((features[0],features[2]))
+    >>> features  = np.array([[ 1.9,2.3],
+    ...                      [ 1.5,2.5],
+    ...                      [ 0.8,0.6],
+    ...                      [ 0.4,1.8],
+    ...                      [ 1.0,1.0]])
```
Comment (tupui) [inline scipy/cluster/vq.py:291]:
```suggestion
    ...                       [ 1.5,2.5],
    ...                       [ 0.8,0.6],
    ...                       [ 0.4,1.8],
    ...                       [ 1.0,1.0]])
```

File: scipy/cluster/vq.py [hunk 3]
```diff
@@ -406,18 +407,18 @@ def kmeans(obs, k_or_guess, iter=20, thresh=1e-5, check_finite=True,
...
-    ...                    [ 1.0,1.0]])
+    >>> features  = np.array([[ 1.9,2.3],
+    ...                      [ 1.5,2.5],
+    ...                      [ 0.8,0.6],
+    ...                      [ 0.4,1.8],
+    ...                      [ 0.1,0.1],
+    ...                      [ 0.2,1.8],
+    ...                      [ 2.0,0.5],
+    ...                      [ 0.3,1.5],
+    ...                      [ 1.0,1.0]])
```
Comment (tupui) [inline scipy/cluster/vq.py:421]:
```suggestion
    ...                       [ 1.5,2.5],
    ...                       [ 0.8,0.6],
    ...                       [ 0.4,1.8],
    ...                       [ 0.1,0.1],
    ...                       [ 0.2,1.8],
    ...                       [ 2.0,0.5],
    ...                       [ 0.3,1.5],
    ...                       [ 1.0,1.0]])
```

**Review by tupui — CHANGES_REQUESTED** [review]
Thank you @njj67229! A few suggestions 😃

**Review by tupui — APPROVED** [review]
LGTM, thanks @njj67229 for the PR 🚀

**Pull request merged**